### ISSUE TITLE
[Feature Request] Block dashboard access for ip in multiple failed attempts

### ISSUE THREAD

**Issue by JoseGalRe** [AdguardTeam/AdGuardHome#863]
Hi guys!, will it be possible to add in the future an option to block the ip that try to access the force to the dashboard? it could be useful in vps where anyelse can try to make brute force.
something similar to fail2ban.

also have the possibility to unblock the ip from cli or the yaml file.

**Comment by ameshkov** [comment]
Makes sense, thank you for the feature request!

**Comment by ArchiveBase** [comment]
Same https://github.com/AdguardTeam/AdGuardHome/issues/675 and related enhancement https://github.com/AdguardTeam/AdGuardHome/issues/1006

**Comment by RainmakerRaw** [comment]
I found this searching before I posted an issue. Before (If) this is ever implemented, can anyone tell me what string to use in Fail2Ban to catch unauthorized logins on AdGuard Home? I checked the log but it only gives generic output like this:

`Auth: invalid user name or password: name='admin'`

I need it to list the offending IP as well, obviously, else F2B can't work. Can I increase the log level to include these, or is there another way (or even another logfile) I've missed? Thanks in advance.
